[2023-09-01T06:23:54.073+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_test.task1 manual__2023-09-01T06:23:52.831793+00:00 [queued]>
[2023-09-01T06:23:54.087+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_test.task1 manual__2023-09-01T06:23:52.831793+00:00 [queued]>
[2023-09-01T06:23:54.089+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-09-01T06:23:54.108+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task1> on 2023-09-01 06:23:52.831793+00:00
[2023-09-01T06:23:54.115+0000] {standard_task_runner.py:57} INFO - Started process 9766 to run task
[2023-09-01T06:23:54.122+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'bash_test', 'task1', 'manual__2023-09-01T06:23:52.831793+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/test_dags2.py', '--cfg-path', '/tmp/tmp91hpwl2u']
[2023-09-01T06:23:54.125+0000] {standard_task_runner.py:85} INFO - Job 48: Subtask task1
[2023-09-01T06:23:54.215+0000] {task_command.py:415} INFO - Running <TaskInstance: bash_test.task1 manual__2023-09-01T06:23:52.831793+00:00 [running]> on host 8ef33824b8b6
[2023-09-01T06:23:54.361+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bash_test' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2023-09-01T06:23:52.831793+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-01T06:23:52.831793+00:00'
[2023-09-01T06:23:54.365+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-01T06:23:54.369+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/ && python src/crawling.py -s hankook -c politics']
[2023-09-01T06:23:54.387+0000] {subprocess.py:86} INFO - Output:
[2023-09-01T06:23:54.590+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2023-09-01T06:23:54.595+0000] {subprocess.py:93} INFO -   File "src/crawling.py", line 2, in <module>
[2023-09-01T06:23:54.597+0000] {subprocess.py:93} INFO -     sys.append('.')
[2023-09-01T06:23:54.599+0000] {subprocess.py:93} INFO - AttributeError: module 'sys' has no attribute 'append'
[2023-09-01T06:23:54.601+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-09-01T06:23:54.643+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-01T06:23:54.673+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=bash_test, task_id=task1, execution_date=20230901T062352, start_date=20230901T062354, end_date=20230901T062354
[2023-09-01T06:23:54.703+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 48 for task task1 (Bash command failed. The command returned a non-zero exit code 1.; 9766)
[2023-09-01T06:23:54.742+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-01T06:23:54.775+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
